admin/horace_on.m.template

```diff
@@ -70,7 +90,7 @@ function her_path = build_default_her_path(hor_path)
 if strcmp(hor_folder,'horace_core')
     her_path = fullfile(fp,'herbert_core');
 else
-    her_path = fullfile(hor_path,'herbert_core');    
+    her_path = fullfile(hor_path,'herbert_core');
```

Comment:
If this is an external herbert, we don't want `her_path` defined from `hor_path`, it should be taken from where `herbert_init` is.

Reply:
its exactly what the code doing. Its not building Herbert path from Horace if Herbert is independent